.github/workflows/spelling.yml

```diff
@@ -73,15 +75,34 @@ jobs:
           git commit -m 'stub commit -- includes submodules'
 
       - name: Check Spelling
-        uses: Lombiq/GitHub-Actions/.github/actions/spelling@dev
+        id: check-spelling-action
+        uses: Lombiq/GitHub-Actions/.github/actions/spelling@issue/OSOE-525
         with:
           dictionary-source-prefixes: ${{ inputs.dictionary-source-prefixes }}
           extra-dictionaries: ${{ inputs.extra-dictionaries }}
           config: ${{ inputs.config }}
           spell-check-this: ${{ inputs.spell-check-this }}
+          post-comment: 0
 
       - name: Cancel Workflow on Failure
         if: failure() && inputs.cancel-workflow-on-failure == 'true'
         uses: Lombiq/GitHub-Actions/.github/actions/cancel-workflow@dev
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  comment-pr:
+    name: Comment (PR)
+    runs-on: ubuntu-22.04
+    needs: check-spelling
+    permissions:
+      pull-requests: write
+    if: always() && needs.check-spelling.outputs.followup && github.event_name == 'pull_request'
+    steps:
+      - name: Comment (PR)
+        uses: Lombiq/GitHub-Actions/.github/actions/spelling@issue/OSOE-525
+        with:
+          dictionary-source-prefixes: ${{ inputs.dictionary-source-prefixes }}
+          extra-dictionaries: ${{ inputs.extra-dictionaries }}
+          config: ${{ inputs.config }}
+          spell-check-this: ${{ inputs.spell-check-this }}
```

Comment:
All these are needed in this job as well when posting/hiding comment?

Reply:
Good question - let me try testing without these extra inputs.

Reply:
Testing in a private repo, I confirmed that these extra inputs are not necessary. I've pushed a new commit.